packages/mint-components/src/components/sqm-reward-exchange-list/sqm-reward-exchange-list.tsx

```diff
@@ -0,0 +1,214 @@
+import { Component, h, Host, Prop, State } from "@stencil/core";
+import { withHooks } from "@saasquatch/stencil-hooks";
+import {
+  ExchangeState,
+  Stages,
+  useRewardExchangeList,
+} from "./useRewardExchangeList";
+import {
+  RewardExchangeViewProps,
+  RewardExchangeView,
+} from "./sqm-reward-exchange-list-view";
+import { isDemo } from "@saasquatch/component-boilerplate";
+import deepmerge from "deepmerge";
+import { DemoData } from "../../global/demo";
+import { getProps } from "../../utils/utils";
+
+/**
+ * @uiName Reward Exchange List
+ */
+@Component({
+  tag: "sqm-reward-exchange-list",
+  assetsDirs: ["assets"],
+  shadow: true,
+})
+export class SqmRewardExchangeList {
```

Comment:
This component has a lot of similar sounding props around errors and text, some prop descriptions are needed to give context to where these props are used in the experience